python/src/main/resources/bootstrap.py

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#PYTHON 2 / 3 comptability :
+# bootstrap.py must be runnable with Python 2 and 3
+
+#Remove interactive mode displayhook
+import sys
+sys.displayhook = lambda x: None
+
+
+# Set the signal handler
+import signal
+def intHandler(signum, frame):
+ print ("Paragraph interrupted")
+ raise KeyboardInterrupt()
+
+signal.signal(signal.SIGINT, intHandler)
+
+
+
+def help():
```

Comment:
should this be in the python side? would it be easier to manage if this is checked on the JVM side like a special command?